mopro-core/build.rs

```diff
@@ -375,12 +245,7 @@ fn main() -> color_eyre::eyre::Result<()> {
         // Otherwise, we default to Circom circuit
         println!("cargo:warning=Building Circom circuit...");
 
-        build_circom_circuit(&circuit_dir_path, circuit_name)?;
-
-        #[cfg(feature = "build-native-witness")]
-        build_witness_graph()?;
-
-        build_dylib(&config)?;
+        transpile_wasm(String::from("./examples"));
```

Comment:
what does the `./examples` here mean?

Reply:
It's referencing the directory `mopro-core/examples`. It's telling `rust-witness` to look in that directory (recursively) for wasm files to transpile and link to the rust build. Any wasm files it transpiles will be available in rust for execution as C code.

Reply:
I think it is a little bit confusing
maybe you can add some comment messages above